lnbits/wallets/breez.py

```diff
@@ -0,0 +1,177 @@
+
+            breez_sdk_working_dir = Path(settings.lnbits_data_folder, "breez-sdk")
+            breez_sdk_working_dir.mkdir(parents=True, exist_ok=True)
+            self.config.working_dir = breez_sdk_working_dir.absolute().as_posix()
+
+            seed = breez_sdk.mnemonic_to_seed(settings.breez_greenlight_seed)
+            self.sdk_services = breez_sdk.connect(self.config, seed, SDKListener())
+
+        async def cleanup(self):
+            self.sdk_services.disconnect()
+
+        async def status(self) -> StatusResponse:
+            try:
+                node_info: breez_sdk.NodeState = self.sdk_services.node_info()
+            except Exception as exc:
+                return StatusResponse(f"Failed to connect to breez, got: '{exc}...'", 0)
+
+            return StatusResponse(None, int(node_info.max_payable_msat))
+
+        async def create_invoice(
+            self,
+            amount: int,
+            memo: Optional[str] = None,
+            description_hash: Optional[bytes] = None,
+            unhashed_description: Optional[bytes] = None,
+            **kwargs,
+        ) -> InvoiceResponse:
+            # if description_hash or unhashed_description:
+            #     raise Unsupported("description_hash and unhashed_description")
+
+            breez_invoice: breez_sdk.ReceivePaymentResponse = (
+                self.sdk_services.receive_payment(
+                    breez_sdk.ReceivePaymentRequest(
+                        amount * 1000,  # breez uses msat
+                        memo,
+                        preimage=kwargs.get("preimage"),
+                        opening_fee_params=None,
+                    )
+                )
+            )
+
+            return InvoiceResponse(
+                True,
+                breez_invoice.ln_invoice.payment_hash,
+                breez_invoice.ln_invoice.bolt11,
+                None,
+            )
+
+        async def pay_invoice(
+            self, bolt11: str, fee_limit_msat: int
+        ) -> PaymentResponse:
+            invoice = lnbits_bolt11.decode(bolt11)
+
+            try:
+                req = breez_sdk.SendPaymentRequest(bolt11=bolt11)
+                payment: breez_sdk.Payment = self.sdk_services.send_payment(req)
+            except Exception as exc:
+                logger.info(exc)
+                try:
+                    # try to report issue to Breez to improve LSP routing
+                    self.sdk_services.report_issue(
+                        breez_sdk.ReportIssueRequest.PAYMENT_FAILURE(
+                            breez_sdk.ReportPaymentFailureDetails(invoice.payment_hash)
+                        )
+                    )
+                except Exception as ex:
+                    logger.info(ex)
+                # assume that payment failed?
+                return PaymentResponse(
+                    False, None, None, None, f"payment failed: {exc}"
+                )
+
+            assert not payment.pending, "payment is pending"
```

Comment:
The Payment `pending` field has been replaced with Payment `status`. As the FAILED case is handled in the exception catch, it can be either PENDING or COMPLETE. 
```suggestion
            assert payment.status == breez_sdk.PaymentStatus.COMPLETE, "payment is pending"
```